pkg/build/graph/helpers.go

```diff
@@ -76,3 +78,27 @@ func defaultNamespace(value, defaultValue string) string {
 	}
 	return value
 }
+
+// BuildPointsToTag returns the buildConfig that points to the provided imageStreamTag.
+func BuildPointsToTag(g osgraph.Graph, istag graph.Node) (*buildgraph.BuildConfigNode, bool) {
+	for _, bcNode := range g.PredecessorNodesByEdgeKind(istag, BuildOutputEdgeKind) {
+		return bcNode.(*buildgraph.BuildConfigNode), true
```

Comment:
If the purpose of the method is to return the one and only ISTag for a BC, `ImageStreamTagForBuildConfig` seems like a good candidate?